thoth-api/src/model/publication/crud.rs

```diff
@@ -182,19 +185,11 @@ where
     }
 }
 
-impl PublicationValidation for NewPublication {
-    fn can_have_isbn(&self, db: &crate::db::PgPool) -> ThothResult<()> {
-        publication_can_have_isbn(self.work_id, db)
-    }
-}
+impl PublicationValidation for NewPublication {}
 
-impl PublicationValidation for PatchPublication {
-    fn can_have_isbn(&self, db: &crate::db::PgPool) -> ThothResult<()> {
-        publication_can_have_isbn(self.work_id, db)
-    }
-}
+impl PublicationValidation for PatchPublication {}
 
-fn publication_can_have_isbn(work_id: Uuid, db: &crate::db::PgPool) -> ThothResult<()> {
+fn publication_can_have_isbn(work_id: &Uuid, db: &crate::db::PgPool) -> ThothResult<()> {
```

Comment:
I've just realised this function is not actually used anywhere else, so it should just be part of the trait